assets/quick-order-list.css

```diff
@@ -529,7 +466,6 @@ quick-order-list-remove-button:hover .icon-remove {
 
   .quick-order-list__table td {
     padding-top: 1.6rem;
-    vertical-align: middle;
```

Comment:
This is needed to ensure this doesnt happen

https://screenshot.click/28-12-do10b-tbgf3.mp4